web/modules/custom/portland/modules/portland_leaftlet_maps/encoding.js

```diff
@@ -1,3313 +0,0 @@
-// This is free and unencumbered software released into the public domain.
```

Comment:
I can't recall why these two files were added. Do you know how they were used?

Reply:
The commit message had a bit more detail, but basically they were a polyfill for the JS TextEncoder API for older browsers like Internet Explorer. 

The TextEncoder API is used when a .zip geomap is loaded by the Leaflet map.

Given that IE is long dead and not supported by the City, I removed them to save a lot of bandwidth when a page with leaflet is loaded.